cmd/gapic-showcase/run.go

```diff
@@ -60,6 +67,31 @@ func init() {
 				grpc.StreamInterceptor(observerRegistry.StreamInterceptor),
 				grpc.UnaryInterceptor(observerRegistry.UnaryInterceptor),
 			}
+
+			// load mutual TLS cert/key and root CA cert
+			if tlsCaCert != "" || tlsCert != "" || tlsKey != "" {
+		    		x509KeyPair, err := tls.LoadX509KeyPair(tlsCert, tlsKey)
```

Comment:
Please rename `x509KeyPair`  to just `keyPair`

Reply:
done